modify-code-and-markdown.ipynb

```diff
@@ -32,10 +32,10 @@
     "import apache_beam as beam\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    added some stuff
</pre><p>Commenting on this thing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/smith-kyle/notebook-diff-tests/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>